frontend/src/components/InfoBox/index.js

```diff
@@ -197,42 +228,38 @@ class InfoBox extends Component {
                   </a>
                 </span>
               </div>
-
-              {!!this.props.data.aliases.length && (
+              {Boolean(aliases.length) && (
                 <div className="crosslist">
-                  Also:
-                  {this.props.data.aliases.map((cls, i) => [
+                  Also:{' '}
+                  {aliases.map((cls, i) => [
                     i > 0 && ', ',
                     <Link key={i} to={`/course/${cls}`}>
                       {cls}
                     </Link>,
                   ])}
                 </div>
               )}
-
-              <p className="subtitle">{this.props.data.name}</p>
-
-              {this.props.data.notes.map((note, i) => (
+              <p className="subtitle">{name}</p>
+              {notes.map((note, i) => (
                 <div key={i} className="note">
                   <i className="fa fa-thumbtack" /> {note}
                 </div>
               ))}
-
-              {this.props.type === 'course' && this.props.liveData && (
+              {pageType === 'course' && liveData && (
                 <Tags
-                  {...this.props.liveData}
-                  data={this.props.data}
-                  existing_instructors={Object.values(
-                    this.props.data.instructors
-                  ).map(a => a.name)}
+                  {...liveData}
+                  data={data}
+                  existing_instructors={Object.values(instructors).map(
```

Comment:
would prefer camelCase, but this can come at a later PR

Reply:
fixed this one for now.